app/views/layouts/_sorting_bar.haml

```diff
@@ -0,0 +1,34 @@
+%nav.navbar.bg-light.sorting-nav
+  .mr-auto
+    .dropdown
+      %button.dropbtn
+        Category
+        %i.fa.fa-angle-down
+      / .dropdown-content
```

Comment:
remove commented code

Reply:
commented code is to be used after food categories are implemented. 
they're commented while the category implementation is postponed.